chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+\textit{Power calculation} and \textit{randomization inference}
 are the main methods by which these probabilities of error are assessed.
-These analytical dimensions are particularly important in the initial phases of development research --
-typically conducted well before any actual field work occurs --
-and often have implications for feasibility, planning, and budgeting.
+These analyses are particularly important in the initial phases of development research --
+typically conducted before any actual field work occurs --
+and have implications for feasibility, planning, and budgeting.
 
-In this chapter, we first cover the necessary practices to ensure that random processes are reproducible.
-We next turn to how to implement sampling and randomized assignment,
-both for simple, uniform probability cases, and more complex designs,
-such as those that require clustering or stratification.
-We include code examples so the guidance is concrete and applicable.
-The last section discusses power calculations and randomization inference,
-and how both are important tools to critically and quantitatively assess different
-sampling and randomization designs and to make optimal choices when planning studies.
+%-----------------------------------------------------------------------------------------------
+\subsection{Randomizing sampling and treatment assignment}
+
+% sampling universe: the master dataset
+\textbf{Sampling} is the process of randomly selecting units-of-observation
+from a list of individuals to create a representative or statistically similar sub-sample.\index{sampling}
+This can, for example, select a subset from all eligible units
+to be included in data collection when the cost of collecting data on everyone is prohibitive.\sidenote{
+	\url{https://dimewiki.worldbank.org/Sample\_Size\_and\_Power\_Calculations}}
+But it can also select a sub-sample of your observations to test a computationally heavy process 
+before running it on the full data.
+\textbf{Randomized treatment assignment} is the process of assigning observations into treatment arms,
+and this is central to experimental research design.
+Most of the code processes used for randomized assignment are the same as those used for sampling,
+since it is also a process of splitting a list of observations into groups.
+Where sampling determines whether a particular individual
+will be observed at all in the course of data collection,
+randomized assignment determines if each individual will be observed
+as a treatment observation or used as a counterfactual.
+The list of units to sample or assign from may be called a \textbf{sampling universe},
+a \textbf{listing frame}, or something similar.
+This list should always be your \textbf{master dataset} when possible.
+The rare exceptions when master datasets cannot be used is when sampling must be done in real time,
+for example, randomly sampling patients as they arrive at a health facility.
```

Comment:
add that in this case it will be important to document as much as possible so that treatment assignment can be added to master after

Reply:
done in fc3bf627c81346ad5c8b35a646d7faef8d6626b4